AGGREGATE.md

```diff
@@ -520,6 +521,26 @@ contribution to any one bucket). Care should be taken to ensure that either:
 See [issue 249](https://github.com/WICG/conversion-measurement-api/issues/249)
 for more details.
 
+### Choosing among aggregation services
+
+The server can respond with an optional header
+`Attribution-Reporting-Alternative-Aggregation-Mode` which accepts a string
+value.
+
+```http
+Attribution-Reporting-Register-Aggregatable-Source: [{....}]
+Attribution-Reporting-Alternative-Aggregation-Mode: "experimental-poplar"
```

Comment:
Is this meant to include all DPF-based aggregation? I think "poplar" unnecessarily narrows it down to the heavy-hitters application that poplar solves. Maybe "experimental-dpf", "experimental-mpc" or "experimental-secret-shares"?

Reply:
I think this is meant as one example of a specific protocol, rather than expecting all services which use DPFs from sharing a common setting. I support leaving as-is for now